db/schema.rb

```diff
@@ -11,156 +11,147 @@
-
   create_table "admin_users", force: :cascade do |t|
-    t.string   "email",                  default: "", null: false
-    t.string   "encrypted_password",     default: "", null: false
-    t.string   "reset_password_token"
+    t.string   "email",                  :default=>"", :null=>false, :index=>{:name=>"index_admin_users_on_email", :unique=>true}
+    t.string   "encrypted_password",     :default=>"", :null=>false
+    t.string   "reset_password_token",   :index=>{:name=>"index_admin_users_on_reset_password_token", :unique=>true}
     t.datetime "reset_password_sent_at"
     t.datetime "remember_created_at"
-    t.integer  "sign_in_count",          default: 0,  null: false
+    t.integer  "sign_in_count",          :default=>0, :null=>false
     t.datetime "current_sign_in_at"
     t.datetime "last_sign_in_at"
     t.string   "current_sign_in_ip"
     t.string   "last_sign_in_ip"
-    t.datetime "created_at",                          null: false
-    t.datetime "updated_at",                          null: false
+    t.datetime "created_at",             :null=>false
+    t.datetime "updated_at",             :null=>false
   end
 
-  add_index "admin_users", ["email"], name: "index_admin_users_on_email", unique: true
-  add_index "admin_users", ["reset_password_token"], name: "index_admin_users_on_reset_password_token", unique: true
-
   create_table "camps", force: :cascade do |t|
-    t.string   "name",                                                     limit: 64,                   null: false
-    t.string   "subtitle",                                                 limit: 255,                  null: false
-    t.string   "contact_email",                                            limit: 64
-    t.string   "contact_name",                                             limit: 64,                   null: false
-    t.string   "contact_phone",                                            limit: 64
-    t.text     "description",                                              limit: 4096
-    t.text     "electricity",                                              limit: 255
-    t.text     "light",                                                    limit: 512
-    t.text     "fire",                                                     limit: 512
-    t.text     "noise",                                                    limit: 255
-    t.text     "nature",                                                   limit: 255
-    t.text     "moop",                                                     limit: 512
-    t.text     "plan",                                                     limit: 1024
-    t.text     "cocreation",                                               limit: 1024
-    t.text     "neighbors",                                                limit: 512
-    t.text     "budgetplan",                                               limit: 1024
+    t.string   "name",                                                     :limit=>64, :null=>false
```

Comment:
@sh0oki Why does your schema looks like this? isn't it the old rails methods? the :limit instead of limit: ?